PolyEngine/Standalone/macOS/Standalone.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,275 @@
+// !$*UTF8*$!
```

Comment:
This file should probably be added to your `.git/info/exclude` or maybe even to `.gitignore`